cmd/blockchaincmd/change_weight.go

```diff
@@ -64,6 +64,8 @@ The L1 has to be a Proof of Authority L1.`,
 	cmd.Flags().Uint64Var(&newWeight, "weight", 0, "set the new staking weight of the validator")
 	cmd.Flags().BoolVarP(&useEwoq, "ewoq", "e", false, "use ewoq key [fuji/devnet only]")
 	cmd.Flags().StringVar(&nodeIDStr, "node-id", "", "node-id of the validator")
+	cmd.Flags().StringVar(&publicKey, "bls-public-key", "", "set the BLS public key of the validator to add")
+	cmd.Flags().StringVar(&pop, "bls-proof-of-possession", "", "set the BLS proof of possession of the validator to add")
```

Comment:
pls change the description to something related to the change weight command, eg "BLS proof of possession of the validator that is to change weight"

Reply:
pls change the description to something related to the change weight command, eg "BLS proof of possession of the validator that is to change weight"